sda/internal/schema/schema.go

```diff
@@ -179,3 +179,8 @@ type SyncMetadata struct {
 type Metadata struct {
 	Metadata interface{}
 }
+
+type KeyhashInsertion struct {
+	Hash        string `json:"hash"`
```

Comment:
```suggestion
	C4ghPubKey        string `json:"c4ghPubKey"`
```
This should not be the key hash but the full public key as a base64 encoded string, i.e `cat c4gh.pub ¦ base64 -w0`
We should also do validation so that we don't  accept things like empty strings.